src/components/UI/Buttons/Button.tsx

```diff
@@ -2,16 +2,37 @@ import tw from 'twin.macro'
 import styled from "@emotion/styled"
 import theme from "../Utils/theme"
 import { css } from "@emotion/react"
-import { isDark } from ".."
 import { HTMLMotionProps, motion } from "framer-motion"
 
 
 export interface ButtonProps extends HTMLMotionProps<"button"> {
 	dark?: boolean
+	centered?: boolean
+	fab?: boolean
+	icon?: boolean
+	height?: number
+	width?: number
+	size?: number
 }
 
-const Button = styled(motion.button)(({ dark }: ButtonProps) => [
-	tw`rounded-lg block px-4 py-2 mt-2 text-sm font-semibold cursor-pointer border-none bg-transparent`,
+const Button = styled(motion.button)(({ icon, height, size, width, fab, centered, dark }: ButtonProps) => [
```

Comment:
- perhaps there's a way to minimize the CSS within the code?

Reply:
What would you sugest?

Reply:
Either an external CSS file or an external ts file, not sure which option is better.